src/index.js

```diff
@@ -50,6 +51,108 @@ class AEMHeadless {
     this.fetch = this.__getFetch(config.fetch)
   }
 
+  buildQuery (model, itemQuery, args) {
```

Comment:
now that we have QueryBuilder, should we create another flavor of `runQuery` that leverages QB instead of passing the whole query?

Reply:
`runPaginatedQuery` is exactly this.
I've covered all types:
- List
- Paginated
- byPath

The only overhead is that if you don't want pagination, you still have to call `.next()`.

```
        const pathQuery = await aemHeadlessClient.runPaginatedQuery(model, fields, { _path: '/content/dam/wknd-shared/en/magazine/alaska-adventure/alaskan-adventures' })
        const result = pathQuery.next()
```

It's easy to add `another flavour of runQuery`, it will be just a method which calls `runPaginatedQuery` and also executes `.next()` in it.

The biggest problem here is to come up with a name for it :)

Reply:
ok, I can see how it could be done currently, without `runPaginatedQuery`:
```
const qbResult = client.buildQuery(model, fields, args)
const response = await client.runQuery(qbResult.query)
```
is that correct?

Reply:
Yes, it can be used like this also